tests/accelerators/test_ddp.py

```diff
@@ -117,3 +120,35 @@ def setup(self, stage: Optional[str] = None) -> None:
     )
     with pytest.raises(SystemExit):
         trainer.fit(model)
+
+
+@RunIf(min_gpus=2, min_torch="1.8.1", special=True)
+def test_ddp_wrapper(tmpdir):
+    """
+    Test parameters to ignore are carried over for DDP.
+    """
+
+    class WierdModule(torch.nn.Module):
+
+        def _save_to_state_dict(self, destination, prefix, keep_vars):
+            return {"something": "something"}
+
+    class CustomModel(BoringModel):
+
+        def __init__(self):
+            super().__init__()
+            self.wierd_module = WierdModule()
```

Comment:
```suggestion
            self.weird_module = WeirdModule()
```
Idk if it's supposed to Weird or Wired 😂

Reply:
Done !